api/incubator/src/main/java/io/opentelemetry/api/incubator/events/EventBuilder.java

```diff
@@ -5,28 +5,107 @@
 
 package io.opentelemetry.api.incubator.events;
 
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.incubator.logs.AnyValue;
+import io.opentelemetry.api.logs.Severity;
+import io.opentelemetry.context.Context;
 import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.concurrent.TimeUnit;
 
 /** The EventBuilder is used to {@link #emit()} events. */
 public interface EventBuilder {
 
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, String value) {
+    return put(key, AnyValue.of(value));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, long value) {
+    return put(key, AnyValue.of(value));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, double value) {
+    return put(key, AnyValue.of(value));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, boolean value) {
+    return put(key, AnyValue.of(value));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, String... value) {
+    List<AnyValue<?>> values = new ArrayList<>(value.length);
+    for (String val : value) {
+      values.add(AnyValue.of(val));
+    }
+    return put(key, AnyValue.of(values));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, long... value) {
+    List<AnyValue<?>> values = new ArrayList<>(value.length);
+    for (long val : value) {
+      values.add(AnyValue.of(val));
+    }
+    return put(key, AnyValue.of(values));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, double... value) {
+    List<AnyValue<?>> values = new ArrayList<>(value.length);
+    for (double val : value) {
+      values.add(AnyValue.of(val));
+    }
+    return put(key, AnyValue.of(values));
+  }
+
+  /** Put the given {@code key} and {@code value} in the payload. */
+  default EventBuilder put(String key, boolean... value) {
+    List<AnyValue<?>> values = new ArrayList<>(value.length);
+    for (boolean val : value) {
+      values.add(AnyValue.of(val));
+    }
+    return put(key, AnyValue.of(values));
+  }
```

Comment:
The test coverage on these default methods comes from the SDK. Do we care about having direct coverage from the API itself?